docker_push.sh

```diff
@@ -58,14 +58,14 @@ else
     export suffix=""
 
     # if we're in prod mode, we'll push the latest image.
-    docker tag $DOCKER_IMAGE $DOCKER_REGISTRY/$DOCKER_NAME:latest
-    docker push $DOCKER_REGISTRY/$DOCKER_NAME:latest
+    docker tag $DOCKER_IMAGE $DOCKER_REPO/$DOCKER_NAME:latest
+    docker push $DOCKER_REPO/$DOCKER_NAME:latest
 fi
 
-docker tag $DOCKER_IMAGE $DOCKER_REGISTRY/$DOCKER_NAME:$git_sha$suffix
-docker push $DOCKER_REGISTRY/$DOCKER_NAME:$git_sha$suffix
+docker tag $DOCKER_IMAGE $DOCKER_REPO/$DOCKER_NAME:$git_sha$suffix
+docker push $DOCKER_REPO/$DOCKER_NAME:$git_sha$suffix
 
 git tag --points-at HEAD | while read -r tag ; do
-    docker tag $DOCKER_IMAGE $DOCKER_REGISTRY/$DOCKER_NAME:$tag$suffix
-    docker push $DOCKER_REGISTRY/$DOCKER_NAME:$tag$suffix
+    docker tag $DOCKER_IMAGE $DOCKER_REPO/$DOCKER_NAME:$tag$suffix
```

Comment:
This looks a bit weird to me. Before with DOCKER_REGISTRY=auroradevacr.azurecr.io and DOCKER_NAME=gordo-components/gordo-model-builder this would expand to `auroradevacr.azurecr.io/gordo-components/gordo-model-builder`, but now it expands to `gordo-components/gordo-model-builder`. I see the variable `DOCKER_REGISTRY` set in travis/circleci, but I dont see how it is used.

Reply:
When doing `docker login` in at the start to `DOCKER_REGISTRY` it will automatically push to that one. It is also set in the `push-images` step here for circleci